packages/client/hmi-client/src/components/data-explorer/search-bar.vue

```diff
@@ -88,7 +127,35 @@ watch(defaultText, (newText) => {
 	right: 0.5rem;
 }
 
+.clear-search-terms {
+	background-color: transparent;
+	padding: 0.5rem;
+	border-radius: 1rem;
+}
+
+.clear-search-terms:hover {
+	color: var(--un-color-body-text-primary);
```

Comment:
Please use the values within the new theme CSS properties:
https://github.com/DARPA-ASKEM/TERArium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L834